xml/System/GCMemoryInfo.xml

```diff
@@ -316,7 +316,7 @@ The memory after `OBJ_D` is not considered part of the `FragmentedBytes` but is
         <summary>Gets the physical memory load when the last garbage collection occurred.</summary>
         <value>The physical memory load, in bytes, when the last garbage collection occurred.</value>
         <remarks>
-          <para>On Windows, memory load is a field in the [MEMORYSTATUS structure](/windows/win32/api/winbase/ns-winbase-memorystatus). It's a number between 0 and 100 that specifies the approximate percentage of physical memory that's in use (0 indicates no memory use and 100 indicates full memory use). Corresponding information is given on other operating systems.</para>
+          <para>On Windows, memory load is a field in the [MEMORYSTATUS structure](https://learn.microsoft.com/en-us/windows/win32/api/winbase/ns-winbase-memorystatus) in bytes that denotes the physical memory that's in use if the process isn't running in a container with a memory limit. Otherwise, it's either the cgroup's physical memory in use or the current process' working set if we are on Windows or somehow can't obtain it from Linux. Corresponding information is given on other operating systems.</para>
```

Comment:
```suggestion
          <para>On Windows, memory load is a field in the [MEMORYSTATUS structure](https://learn.microsoft.com/en-us/windows/win32/api/winbase/ns-winbase-memorystatus) in bytes that denotes the physical memory that's in use if the process isn't running in a container with a memory limit. For other operating systems, or if the process is running on Windows in a container with a memory limit, memory load is either the cgroup's physical memory in use or the current process's working set.</para>
```

Reply:
Always add an apostrophe followed by `s` to form the possessive of a singular noun, even if it ends in `s` (see https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28753).